Example/AsyncReactorExample/Features/Repository/Detail/RepositoryDetailView.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  RepositoryDetailView.swift
+//  AsyncReactorExample
+//
+//  Created by Ahmet Bozkan on 22.05.23.
+//
+
+import SwiftUI
+import AsyncReactor
+
+struct RepositoryDetailView: View {
+    @EnvironmentObject
+    private var reactor: RepositoryDetailReactor
+    
+    var repository: Repository
+    
+    @ActionBinding(RepositoryDetailReactor.self, keyPath: \.sheetPresented, action: RepositoryDetailReactor.Action.setSheetPresented)
+    private var sheetPresented: Bool
+    
+    var body: some View {
+        NavigationView {
```

Comment:
You should not have a `NavigationView` here in the detail view. Since you are pushing this onto the stack already, you would have a NavigationView inside another NavigationView.